apps/website/src/data/found-animal.ts

```diff
@@ -242,6 +242,192 @@ const data: FoundAnimalFlow = {
+                      "This is likely a juvenile squirrel, you do not need to intervene.",
+                      ...macros.general.leave("the squirrel"),
+                    ],
+                  },
+                },
+                {
+                  name: "No",
+                  flow: {
+                    prompt: [
+                      "Is the squirrel alone?",
+                    ],
+                    options: [
+                      {
+                        name: "Yes",
+                        flow: {
+                          prompt: [
+                            "You've likely found an infant squirrel, who needs to be guided back to their mother.",
+                            "Place uncooked rice or bird seed in a sock and warm in the microwave for 20-30 seconds. Then, wrap the sock in a soft towel and place it with the baby squirrel in an open container (e.g. a box) reheating the rice every two hours.",
+                            "Do not attempt to feed or give water to the squirrel, as this may lead to it imprinting on humans and becoming dependent on them.",
+                            "Return the squirrel to its nesting tree (usually in the immediate area where the squirrel was found). If you're unsure of the right tree, or if the nest was destroyed, choose a tree closest to where the squirrel was found.",
+                            "Observe for 6-8 hours, reheating the rice every two hours",
+                            "Has the mother returned?",
+                          ],
+                          options: [
+                            {
+                              name: "Yes",
+                              flow: {
+                                prompt: macros.general.leave("the squirrel"),
+                              },
+                            },
+                          ],
+                        },
+                      },
+                      {
+                        name: "No",
+                        flow: {
+                          prompt: macros.general.rehab(
+                            "the squirrel",
+                            "If you are sure the parents are not nearby, and do not return within a few hours"),
+                        },
+                      },
+                    ],
+                  },
+                },
+              ],
+            },
+          },
+        ],
+      },
+    },
+    {
+      name: "Raccoon",
+      flow: {
+        prompt: [
+          "Does the raccoon appear to be sick or injured?",
+        ],
+        options: [
+          {
+            name: "Yes",
+            flow: {
+              prompt: [
+                "Is the raccoon an adult or a baby?",
+                "Baby raccoons can range from 4-14 inches long. Newborns eyes and ears are closed, babies are extremely vocal, growling, hissing, and snorting if disturbed. A full grown adult can be up to 3 feet long. Male's tend to be larger than females",
```

Comment:
```suggestion
                "Baby raccoons are normally between 4 and 14 inches in length, whereas a fully grown adult can be up to 3 feet long. Babies are often extremely vocal, growling, hissing, and snorting if disturbed.",
```
Rewording to read better, and removing the male/female context as we're not asking the user about that.